src/three/TilesRenderer.js

```diff
@@ -807,13 +802,14 @@ export class TilesRenderer extends TilesRendererBase {
 
 					const pixelSize = info.pixelSize;
 					error = tile.geometricError / ( pixelSize * invScale );
+					// TODO: distance not updated while in orthographic views
```

Comment:
I'm not really sure what to do about this one, not setting a distance here *seems* like it shouldn't introduce different behavior from current builds with the distance based parse priority sorting, since the values will be equal.

If that is still a desired behavior, I'm wondering if we add another level in the default sort function which falls through to __depthFromRenderedParent when the distances are equal to match previous priorityFunction behavior?

Again, only an edge case with sorting the tiles at geometricError: 0, as all other tiles will end up sorted based on the error metric

Reply:
Not considering the distance when only an orthographic camera is used makes sense to me. I agree I think we should prioritize the shallowest tile if the distances are the same. At least we need to return 0 if the distances are the same because `Infinity - Infinity = NaN`.